ci-operator/config/stolostron/clusterlifecycle-state-metrics/stolostron-clusterlifecycle-state-metrics-main.yaml

```diff
@@ -2,7 +2,7 @@ base_images:
   base:
     name: ubi-minimal
     namespace: ocp
-    tag: "8"
+    tag: "9"
```

Comment:
Should change the file .stolostron-clusterlifecycle-state-metrics-backplane-2.7.yaml as well.